test/git.spec.js

```diff
@@ -66,7 +66,7 @@ function mock ({ bump, changelog, tags }) {
     }
   }))
 
-  mockery.registerMock('git-semver-tags', function (cb) {
+  mockery.registerMock('git-semver-tags', function (_, cb) {
```

Comment:
@jahead this is looking good to me, but any chance we could add a test that covers the additional functionality?

Reply:
Yea mate, I"ll do it tonight. 

I was thinking how to test this

Reply:
@bcoe  Hey so I added some tests here and tested tagPrefix in general, as there didn't seem to be any tests already that could see. 

I'm not overly happy with is this as ` git-semver-tags` actually is the one filtering the tags, and due to the mock we won't know if  `git-semver-tags` changes will break the logic here. But the tests cover the use case in #702 

resolve if you're happy